### PR TITLE
test: add more checks on interaction tests

### PR DESCRIPTION
**Pull request by EmilyBonar**
## Description
There has been some flakiness with this test on CircleCI. I added in more checks to make sure things are rendering correctly, and I'm going to run this test repeatedly on CI to see if I can't track down any more bugs.

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by EmilyBonar** [comment]
I'm running into more test failures, this time on TagList. I'm going to add that to the scope of this PR.

File: webui/react/src/components/TagList.test.tsx [hunk 1]
```diff
@@ -66,7 +63,9 @@ describe('TagList', () => {
     const input = screen.getByLabelText(ARIA_LABEL_INPUT);
     userEvent.type(input, `${rename}{enter}`);
 
-    const resultTags = clone(initTags).filter((tag: string) => tag !== renameTag);
+    //screen.debug();
+
```
Comment (hkang1) [inline webui/react/src/components/TagList.test.tsx:67]: **chore:** remove

File: webui/react/src/components/TagList.tsx [hunk 2]
```diff
@@ -125,16 +95,14 @@ const EditableTagList: React.FC<Props> = (
...
                 key={tag}
                 ref={editInputRef}
                 size="small"
                 style={{ width: inputWidth }}
-                value={editInputValue}
                 width={inputWidth}
-                onBlur={handleEditInputConfirm}
-                onChange={handleEditInputChange}
-                onFocus={handleEditInputFocus}
-                onPressEnter={handleEditInputConfirm}
```
Comment (hkang1) [inline webui/react/src/components/TagList.tsx:137]: nice 👍 much cleaner! Tried out the updated taglist via preview, this seems to work without any issues too.

File: webui/react/src/hooks/useModal/useModalCustomizeColumns.test.tsx [hunk 3]
```diff
@@ -104,50 +114,64 @@ describe('useCustomizeColumnsModal', () => {
   });
 
   it('hides column', async () => {
-    const { view } = await setup();
+    await setup();
 
     //waiting for modal to render
```
Comment (hkang1) [inline webui/react/src/hooks/useModal/useModalCustomizeColumns.test.tsx:119]: **nit:** add space between `//` and `waiting`

**Review by hkang1 — APPROVED** [review]
I ran this locally for over 200 times and it did not flake. I've re-ran it on the CI a few times and it was ok as well.

**Pull request merged**